### PR TITLE
Fix empty dirname and empty data.crs.get('init')

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@aashish24 @dorukozturk Please review.

File: gaia/inputs.py [hunk 1]
```diff
@@ -119,10 +120,15 @@ def get_epsg(self):
...
-                    self.epsg = int(crs)
+                if crs:
+                    if ':' in crs:
+                        crs = crs.split(':')[1]
+                    if crs.isdigit():
+                        self.epsg = int(crs)
+                        return self.epsg
+                    # Assume EPSG:4326
+                    self.epsg = 4326
+                    self.data.crs = fiona_crs.from_epsg(4326)
```
Comment (aashish24) [inline gaia/inputs.py]: I would rather pass  self.epsg

Comment (danlipsa) [reply gaia/inputs.py]: Done

**Comment by aashish24** [comment]
thanks @danlipsa can you do the similar for other places as well?

**Comment by danlipsa** [comment]
I found just one more place.

**Comment by aashish24** [comment]
@danlipsa perhaps I am missing something but look at like 117 and just before that. 

self.epsg = 3857

**Comment by danlipsa** [comment]
@aashish24 Done. Let's merge!

**Pull request merged**